app/routes/topics/index.tsx

```diff
@@ -1,12 +1,41 @@
-import { Link } from "@remix-run/react";
+import type { Topic } from "@prisma/client";
+import { Link, useLoaderData } from "@remix-run/react";
+import type { LoaderArgs } from "@remix-run/server-runtime";
+import { json } from "@remix-run/server-runtime";
+import { getTopicListItems } from "~/models/topic.server";
+import { requireUserId } from "~/session.server";
+
+export async function loader({ request }: LoaderArgs) {
+  const userId = await requireUserId(request);
+  const topics = await getTopicListItems({ userId });
```

Comment:
```suggestion
  const topics = await getTopicListItems();
```
if we're doing the whole shared DB idea then users should be able to see all the topics posted, right? and then we can remove the line above as well as the last import & `getTopicListItems` doesn't take an argument then

Reply:
I also refactored this in my PR here: [search params](https://github.com/remixcph/topic-storm/pull/18) so you can also just leave it